platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -141,8 +153,9 @@ class ConfigurationCacheProblemsSummary(
         }
     }
 
-    fun onIncompatibleFeature() {
+    fun onIncompatibleFeature(problem: PropertyProblem) {
         lock.withLock {
+            onProblem(problem, ProblemSeverity.SuppressedSilently)
```

Comment:
For incompatible features, we were adding a problem to the report without letting the summarizer know - now we do report the problem so it is reflected on `totalProblemCount` and `suppressedSilentlyProblemCount`.